lib/rules/no-restricted-imports.js

```diff
@@ -248,26 +273,29 @@ module.exports = {
          */
         function checkNode(node) {
             const importSource = node.source.value.trim();
-            const importNames = node.specifiers ? node.specifiers.reduce((set, specifier) => {
+            const importNames = node.specifiers ? node.specifiers.reduce((map, specifier) => {
+                const specifierKey = specifier.imported ? "imported" : "local";
+                const specifierData = { loc: specifier.loc.start };
```

Comment:
This should be just `specifier.loc`, without `.start`.

Also, it would be nice to have locations in the `invalid` tests (`column` and `endColumn` error properties, `line` if needed).

I think that it also makes sense to report the location of `*` in `reportPathForEverythingImported`.

Reply:
Done!